app/controllers/maintenance_tasks/runs_controller.rb

```diff
@@ -35,6 +35,14 @@ def pause
       redirect_to(task_path(@run.task_name), alert: error.message)
     end
 
+    # Forces paused state to be set, used if stuck
+    def force_pause
```

Comment:
What do you think about keeping the same endpoint, and adding a `pause` method on `MaintenanceTasks::Run` (similar to `#cancel`) that handles either `pausing!` the run or marking it as `paused` in the case where it's stuck?